project/views.py

```
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def custom_404(request, exception):
+    return render(request, '404.html', status=404)
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)